src/styles/_header.scss

```diff
@@ -0,0 +1,66 @@
+@import "variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin-left: 50px;
+
+    .header__img {
+      display: flex;
+      align-self: center;
+    }
+  }
+
+  .nav {
+    margin-right: 50px;
```

Comment:
don't add margins to blocks, make it a padding for your header

Reply:
fix